src/App.js

```diff
@@ -17,12 +17,27 @@ class App extends Component {
       resultDays: 0,
       calculatorInfo: this.decodeToState(),
       // uncomment this if you don't want to have to read from an encoded URL
-      // calculatorInfo: excludeDates,
+      calculatorInfo: excludedDates,
```

Comment:
I'll comment this back out before merging, but also wanted to point out that there was a typo in the variable 🙂

Reply:
thank you!